validator/src/main/java/com/amazon/aoc/fileconfigs/PredefinedExpectedTemplate.java

```diff
@@ -15,18 +15,34 @@
 
 package com.amazon.aoc.fileconfigs;
 
-import lombok.Getter;
+import java.net.URL;
 
-@Getter
-public enum ExpectedTrace implements FileConfig {
+/**
+ * ExpectedTemplate includes all the built-in expected data templates,
+ * which are under resources/expected-data-templates.
+ */
+public enum PredefinedExpectedTemplate implements FileConfig {
```

Comment:
I would prefer merge `LocalPathExpectedTemplate` and `PredefinedExpectedTemplate` into one, and register the predefined templated into local. It looks like predefined is just a subset of local path.

Reply:
there're two differences driving me divide them into two.
1.  PredefinedExceptedTemplate uses different way to get its file path: `GetResource.GetPath()`, as its interface FileConfig defined, the function `getPath` is the key for different implementation.

2. PredefinedExcecptedTemplate is an enum class, i'm not sure how to merge them together, it will be great if you can shed me some lights?

Reply:
Get it. The pain is we need to convert a string to a built-in FileConfig. Let's keep it as what it is, merging two won't make the whole thing easier. Please help update the comment of `PredefinedExpectedTemplate`, I think we need to refine the class name in the java doc.